src/components/BiDCommandInput.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import "./BiDCommandInput.css";
```

Comment:
```suggestion
import "./BiDiCommandInput.css";
```
Small typo in the filenames for this component (js & css)